src/templates/Autocomplete/index.html.hbs

```diff
@@ -0,0 +1,19 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
```

Comment:
We should have their app ID at this stage, right? If yes, we could add a `<link rel="preconnect">` element to further improve the bootstrapped app.

Reply:
We mostly use templates on CodeSandbox, where we modify credentials after templates have been built for bug reproduction. So most of the time, the `<link rel="preconnect">` will not match the actual creds used.

Reply:
if we add this, it should be done for all templates, and maybe even programmatically in the js client